File: codeforces/43/A/43A.java
import java.io.*;
import java.util.*;
// note the class name and the public identifier may differ from one online judge to the other.
public class Main {
public static void main(String [] argv) throws Exception{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int cantidad = Integer.parseInt(br.readLine());
        HashMap<String,Integer> mapa= new HashMap<>(2);
        String[] equipos= new String[2];
        int contador=0;
        for(int i=0;i<cantidad;i++){
            String nombreEquipo=br.readLine();
            if(mapa.containsKey(nombreEquipo)){
                mapa.put(nombreEquipo, mapa.get(nombreEquipo)+1);
            }else{
                mapa.put(nombreEquipo, 1);
                equipos[contador]=nombreEquipo;
                contador++;
            }
            
        }
        if(contador==2){
            String respuesta="";
            int mayor=0;
            for(int i=0;i<2;i++){
                 if(mapa.get(equipos[i])>mayor){
                    mayor=mapa.get(equipos[i]);
                    respuesta=equipos[i];
                }
            }
        System.out.println(respuesta);
        }
        else{
            System.out.println(equipos[0]);
        }
        
    }
}

File: codeforces/43/A/43A.kt
package codeforces

import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val In = BufferedReader(InputStreamReader(System.`in`))
    val lines = In.readLine().toInt()
    var s = ""
    var s2 = ""
    var a = 0
    var b = 0
    for(i in 1..lines) {
        val team = In.readLine()
        if(s.equals("") or s.equals(team) ) {
            s = team
            a++
        }  else {
            s2 = team
            b++
        }
    }
    if (a > b)
        print(s)
    else
        print(s2)
}